### PR TITLE
*: update the note for write conflict and rename the sysbench file

### PR DESCRIPTION
**Pull request by jackysp**
PTAL @lilin90 @shenli @lysu 
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -966,36 +969,44 @@ Cloud TiDB 目前已经在腾讯云、UCloud 上线，都是数据库一级入
 
 ### 9.1 TiDB 自定义报错汇总
 
-#### 9.1.1 ERROR 9001 (HY000) : PD Server Timeout
+#### 9.1.1 ERROR 8005 (HY000) : Write Conflict, txnStartTS is stale
+
+可以检查 `tidb_disable_txn_auto_retry` 是否为 on，如是，将其设置为 off，如已经是 off，将 `tidb_retry_limit` 调大到不再发生该错误。
```
Comment (dcalvin) [inline FAQ.md]:
```suggestion
可以检查 `tidb_disable_txn_auto_retry` 是否为 on。如是，将其设置为 off；如已经是 off，将 `tidb_retry_limit` 调大到不再发生该错误。
```

File: benchmark/how-to-run-sysbench.md [hunk 2]
```diff
@@ -126,17 +116,25 @@ db-driver=mysql
 
 ### 数据导入
 
-MySQL 客户端执行以下 SQL 语句，创建数据库 `sbtest`：
+在数据导入前，需要对 TiDB 进行简单设置。在 MySQL 客户端中执行，
```
Comment (dcalvin) [inline benchmark/how-to-run-sysbench.md]:
```suggestion
在数据导入前，需要对 TiDB 进行简单设置。在 MySQL 客户端中执行如下命令：
```

File: benchmark/how-to-run-sysbench.md [hunk 3]
```diff
@@ -126,17 +116,25 @@ db-driver=mysql
...
 
+```sql
+set global tidb_disable_txn_auto_retry = off;
 ```
+
+然后，退出客户端。TiDB 使用乐观事务模型，当发现并发冲突时，会回滚事务。将 `tidb_disable_txn_auto_retry` 设置为 `off` 会开启事务冲突后的自动重试机制，可以尽可能避免事务冲突报错导致 Sysbench 程序退出的问题。
```
Comment (dcalvin) [inline benchmark/how-to-run-sysbench.md]:
```suggestion
然后退出客户端。TiDB 使用乐观事务模型，当发现并发冲突时，会回滚事务。将 `tidb_disable_txn_auto_retry` 设置为 `off` 会开启事务冲突后的自动重试机制，可以尽可能避免事务冲突报错导致 Sysbench 程序退出的问题。
```

File: dev/reference/error-codes.md [hunk 4]
```diff
@@ -17,13 +17,15 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
...
+| 8004 | 单个事务过大，原因及解决方法请参考[这里](/FAQ.md#出现-transaction-too-large-报错怎么办) |
+| 8005 | 事务在 TiDB 中遇到了写入冲突，原因及解决方法请参考[这里](/FAQ.md#九、故障排除) |
 | 9001 | 请求 PD 超时，请检查 PD Server 状态/监控/日志以及 TiDB Server 与 PD Server 之间的网络 |
 | 9002 | 请求 TiKV 超时，请检查 TiKV Server 状态/监控/日志以及 TiDB Server 与 TiKV Server 之间的网络 |
 | 9003 | TiKV 操作繁忙，一般出现在数据库负载比较高时，请检查 TiKV Server 状态/监控/日志 |
 | 9004 | 当数据库上承载的业务存在大量的事务冲突时，会遇到这种错误，请检查业务代码 |
 | 9005 | 某个 Raft Group 不可用，如副本数目不足，出现在 TiKV 比较繁忙或者是 TiKV 节点停机的时候，请检查 TiKV Server 状态/监控/日志 |
 | 9006 | GC Life Time 间隔时间过短，长事务本应读到的数据可能被清理了，应增加 GC Life Time |
-| 9500 | 单个事务过大，原因及解决方法请参考[这里](/FAQ.md#出现-transaction-too-large-报错怎么办) |
+| 9007 | 事务在 TiKV 中遇到了写入冲突，原因及解决方法请参考[这里](/FAQ.md#九、故障排除) |
```
Comment (dcalvin) [inline dev/reference/error-codes.md]:
```suggestion
| 9007 | 事务在 TiKV 中遇到了写入冲突，原因及解决方法请参考[这里](/FAQ.md#九故障排除) |
```

File: dev/reference/error-codes.md [hunk 5]
```diff
@@ -17,13 +17,15 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
 | 8001 | 请求使用的内存超过 TiDB 内存使用的阈值限制 |
 | 8002 | 带有 `SELECT FOR UPDATE` 语句的事务，在遇到写入冲突时，为保证一致性无法进行重试，事务将进行回滚并返回该错误 |
 | 8003 | `ADMIN CHECK TABLE` 命令在遇到行数据跟索引不一致的时候返回该错误 |
+| 8004 | 单个事务过大，原因及解决方法请参考[这里](/FAQ.md#出现-transaction-too-large-报错怎么办) |
+| 8005 | 事务在 TiDB 中遇到了写入冲突，原因及解决方法请参考[这里](/FAQ.md#九、故障排除) |
```
Comment (dcalvin) [inline dev/reference/error-codes.md]:
```suggestion
| 8005 | 事务在 TiDB 中遇到了写入冲突，原因及解决方法请参考[这里](/FAQ.md#九故障排除) |
```

File: benchmark/how-to-run-sysbench.md [hunk 6]
```diff
@@ -225,8 +223,8 @@ sysbench --config-file=config oltp_read_only --tables=32 --table-size=10000000 r
 | oltp_read_only | 3\*32 | 5066.88 | 81070.16 | 19.42 | 26.20 | 123.41 |
 | oltp_read_only | 3\*64 | 5466.36 | 87461.81 | 34.65 | 63.20 | 231.19 |
 | oltp_read_only | 3\*128 | 6684.16 | 106946.59 | 57.29 | 97.55 | 180.85 |
- 
-![](../media/oltp_read_only.png)
+
+![oltp_read_only](../media/oltp_read_only.png)
```
Comment (dcalvin) [inline benchmark/how-to-run-sysbench.md]:
```suggestion
![oltp_read_only](/media/oltp_read_only.png)
```

File: benchmark/how-to-run-sysbench.md [hunk 7]
```diff
@@ -213,8 +211,8 @@ sysbench --config-file=config oltp_read_only --tables=32 --table-size=10000000 r
 | oltp_update_index | 3\*64 | 18697.17 | 18697.17 | 10.34 | 17.63 | 4539.04 |
 | oltp_update_index | 3\*128 | 20446.81 | 20446.81 | 18.98 | 40.37 | 5394.75 |
 | oltp_update_index | 3\*256 | 23563.03 | 23563.03 | 32.86 | 78.60 | 5530.69 |
- 
-![](../media/oltp_update_index.png)
+
+![oltp_update_index](../media/oltp_update_index.png)
```
Comment (dcalvin) [inline benchmark/how-to-run-sysbench.md]:
```suggestion
![oltp_update_index](/media/oltp_update_index.png)
```

File: benchmark/how-to-run-sysbench.md [hunk 8]
```diff
@@ -201,7 +199,7 @@ sysbench --config-file=config oltp_read_only --tables=32 --table-size=10000000 r
 | point_select | 3\*64 | 195218.54 | 195218.54 | 0.98 | 2.14 | 21.82 |
 | point_select | 3\*128 | 208189.53 | 208189.53 | 1.84 | 4.33 | 31.02 |
 
-![](../media/oltp_point_select.png)
+![oltp_point_select](../media/oltp_point_select.png)
```
Comment (dcalvin) [inline benchmark/how-to-run-sysbench.md]:
```suggestion
![oltp_point_select](/media/oltp_point_select.png)
```

**Comment by jackysp** [comment]
PTAL @dcalvin

File: faq/tidb.md [hunk 9]
```diff
@@ -967,36 +970,44 @@ Cloud TiDB 目前已经在腾讯云、UCloud 上线，都是数据库一级入
...
-```
+```sql
 update mysql.tidb set variable_value='30m' where variable_name='tikv_gc_life_time';
 ```
 
 其中 30m 代表仅清理 30 分钟前的数据，这可能会额外占用一定的存储空间。
 
+#### 9.1.8 ERROR 9007 (HY000) : Write Conflict
+
+可以检查 `tidb_disable_txn_auto_retry` 是否为 on，如是，将其设置为 off，如已经是 off，将 `tidb_retry_limit` 调大到不再发生该错误。
```
Comment (dcalvin) [inline faq/tidb.md]:
```suggestion
可以检查 `tidb_disable_txn_auto_retry` 是否为 on。如是，将其设置为 off；如已经是 off，将 `tidb_retry_limit` 调大到不再发生该错误。
```

**Comment by jackysp** [comment]
PTAL @dcalvin

File: benchmark/how-to-run-sysbench.md [hunk 10]
```diff
@@ -1,22 +1,12 @@
 ---
-title: TiDB 3.0 Beta 版本在 NVMe SSD 上的 Sysbench 测试结果
+title: 如何用 Sysbench 测试 TiDB
```
Comment (dcalvin) [inline benchmark/how-to-run-sysbench.md]:
```suggestion
title: 如何进行 Sysbench 测试（以 TiDB 3.0 Beta 版本为例）
```
Title 需要保持一致

**Comment by jackysp** [comment]
PTAL @dcalvin

**Pull request merged**